content/en/blog/_posts/2023-08-15-pkgs-k8s-io-introduction.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: blog
+title: "pkgs.k8s.io: Introducing Kubernetes community-owned package repositories"
```

Comment:
```suggestion
title: Kubernetes 1.28: "pkgs.k8s.io Introducing Kubernetes Community-owned Package Repositories"
```
Suggestions:
- Title case for title
- The previous blog posts have the version starting the title

Reply:
Hey @reylejano, this is an out of tree change so we intentionally removed the prefix. Is that fine for you, too?

Reply:
Release comms / @bradmccoydev - how do you want to handle this announcement? Do you want it to be part of the v1.28 release comms or separate from it?

I'd assumed that there _is_ a link. You can still set a custom title; if you do, I strongly suggest putting this blog article as the very first or very last post-release blog article for v1.28

Reply:
Hi sure no problem at all, I have updated the Comms Board for it to go out the same day

Reply:
Thanks @bradmccoydev